tst/testinstall/hpc/serialize.tst

```diff
@@ -214,4 +218,23 @@ gap> DeserializeNativeString("\000\377");
 Error, Bad deserialization input
 gap> DeserializeNativeString("\000\205");
 1
+
+#
+# verify that stuff that gets serialized twice is
+# deserialized into the same object each time
+#
+gap> CheckRepeatedSerialization := function(y)
+>   CheckSerializationGeneric([y, y], x->IsIdenticalObj(x[1], x[2]));
+>   CheckSerializationGeneric(rec(a:=y,b:=y), x->IsIdenticalObj(x.a, x.b));
+> end;;
+gap> CheckRepeatedSerialization("abc");
+gap> CheckRepeatedSerialization(2^100);
```

Comment:
Maybe pop in a comment saying this difference is fine, as these are large integers (assuming you plan on leaving the test the same)

Reply:
Actually, this is IMHO not fine, so I'll add a FIXME just like with the other cases. Actually looking at this problem was what caused me to write this PR (as I kept discovering more and more issues); ironically, I've not yet managed to fix the initial problem ;-).